admin/apple-actions/index/class-push.php

```diff
@@ -346,6 +346,10 @@ private function push( $user_id = null ) {
 		// Get the isSponsored setting.
 		$is_sponsored                = (bool) get_post_meta( $this->id, 'apple_news_is_sponsored', true );
 		$meta['data']['isSponsored'] = $is_sponsored;
+		
+		// Get the videoURL suppression setting.
+		// $suppressVideoURL                 = (bool) get_post_meta( $this->id, 'apple_news_suppress_video_url', true );
+		// $meta['data']['suppressVideoURL'] = $suppressVideoURL;
```

Comment:
Doesn't seem to be necessary.  The update pushes even without it.  And adding this code results in an Apple News API error (INVALID_JSON).

Reply:
Yeah, this isn't a valid meta property in the Apple News API, it's only for internal use when deciding whether to set the video URL or the thumbnail URL. This can be removed.